### DISCUSSION TITLE
umi-request 中的 request.use 里如何获取响应头

### DISCUSSION THREAD

**Discussion by georgezouq** [umijs/umi#9142] [Q&A]
需求是记录接口请求时间，如果时间超过2s则进行上报。

一开始是想在拦截器中做，在请求拦截器中记录时间，后在相应拦截器中取出，但发现相应拦截器根本没办法获取请求相关的任何参数。

而后改为在中间件中处理，但是发现中间件中没办法获取相应头，只有最终相应的数据。如果在中间件中获取相应头呢？？？ 或者这种记录请求时间的需求有没有更好的实现方式

ps. 响应头中有记录后端需要的 traceid，记录日志的时候需要用到

**Comment by Lydiayaya** [comment]
你好，请问你解决了这个问题了吗
